### PR TITLE
Add python3-dubins-pip

### PR DESCRIPTION
**Pull request by teddyort**
# Please add the following dependency to the rosdep database.

## Package name:

dubins

## Package Upstream Source:

https://github.com/AndrewWalker/pydubins/

## Purpose of using this:

Use a Dubins planner on a mobile robot.

Distro packaging links:

## Links to Distribution Packages

https://pypi.org/project/dubins/

### CODE REVIEW

File: rosdep/python.yaml
Comment (nuclearsandwich) [inline rosdep/python.yaml:6279]:
```suggestion
python3-dubins-pip:
  debian:
    pip:
      packages: [dubins]
  fedora:
    pip:
      packages: [dubins]
  opensuse:
    pip:
      packages: [dubins]
  osx:
    pip:
      packages: [dubins]
  ubuntu:
    pip:
      packages: [dubins]
```

**Comment by nuclearsandwich** [comment]
@teddyort as long as you're aware that packages with pip dependencies cannot be released on the official ROS build farm this addition looks alright and just needs a second reviewer.

**Review by nuclearsandwich — APPROVED** [review]

**Review by ivanpauno — APPROVED** [review]

**Comment by ivanpauno** [comment]
I will wait until https://github.com/ros/rosdistro/pull/31412#discussion_r765053261 is addressed before merging, LGTM

**Comment by nuclearsandwich** [comment]
@teddyort would you be willing to accept the suggested update so we can merge this one?

**Comment by teddyort** [comment]
Thanks @nuclearsandwich and @ivanpauno I've accepted the suggestion.

**Pull request merged**
